### PR TITLE
[DB] Add db indices to runs and labels

### PR DESCRIPTION
**Pull request by liranbg**
related to https://github.com/mlrun/mlrun/issues/4524

https://jira.iguazeng.com/browse/ML-4903

### CODE REVIEW

File: server/api/db/sqldb/models/models_mysql.py [hunk 1]
```diff
@@ -210,6 +214,7 @@ class Run(Base, mlrun.utils.db.HasStruct):
         __tablename__ = "runs"
         __table_args__ = (
             UniqueConstraint("uid", "project", "iteration", name="_runs_uc"),
+            Index("idx_runs_project_uid", "id", "project", unique=True),
```
Comment (theSaarco) [inline server/api/db/sqldb/models/models_mysql.py:217]:
```suggestion
            Index("idx_runs_project_id", "id", "project", unique=True),
```

File: server/api/db/sqldb/models/models_sqlite.py [hunk 2]
```diff
@@ -208,6 +212,7 @@ class Run(Base, mlrun.utils.db.HasStruct):
         __tablename__ = "runs"
         __table_args__ = (
             UniqueConstraint("uid", "project", "iteration", name="_runs_uc"),
+            Index("idx_runs_project_uid", "id", "project"),
```
Comment (theSaarco) [inline server/api/db/sqldb/models/models_sqlite.py:215]:
```suggestion
            Index("idx_runs_project_id", "id", "project"),
```

File: server/api/migrations_mysql/versions/59061f6e2a87_add_index_migration.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+# revision identifiers, used by Alembic.
+revision = "59061f6e2a87"
+down_revision = "27ed4ecb734c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index("idx_runs_project_uid", "runs", ["id", "project"], unique=True)
```
Comment (theSaarco) [inline server/api/migrations_mysql/versions/59061f6e2a87_add_index_migration.py:33]:
```suggestion
    op.create_index("idx_runs_project_id", "runs", ["id", "project"], unique=True)
```

File: server/api/migrations_mysql/versions/59061f6e2a87_add_index_migration.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        "schedules_v2_labels",
+        ["name", "value"],
+        unique=False,
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index("idx_runs_project_uid", table_name="runs")
```
Comment (theSaarco) [inline server/api/migrations_mysql/versions/59061f6e2a87_add_index_migration.py:93]:
```suggestion
    op.drop_index("idx_runs_project_id", table_name="runs")
```

**Review by theSaarco — CHANGES_REQUESTED** [review]
LGTM - minor change needed.
Also - `api/db/sqldb/helpers.py` imports `_table2cls` - since you modified the rest of imports to get them from `.common`, it might be a good idea to modify there as well (though the function needing it doesn't seem to be used by anyone, might consider removing it).

**Comment by liranbg** [comment]
> LGTM - minor change needed. Also - `api/db/sqldb/helpers.py` imports `_table2cls` - since you modified the rest of imports to get them from `.common`, it might be a good idea to modify there as well (though the function needing it doesn't seem to be used by anyone, might consider removing it).

good point. removing

**Pull request merged**